lib/l10noptimize/parser/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var L10nPropParser = require('./l10n-properties');
+var L20nPropParser = require('./l20n-properties');
+
+exports.parse = function (l10nLib, source) {
+  if (l10nLib === 'l10n.js') {
+    return L10nPropParser.parse(source);
+  } else {
+    return L20nPropParser.parse(source);
```

Comment:
I don't mind the `l10nLib` variable but once we also add the l20n format, this will get more more complex.  Perhaps a cleaner solution would be to directly pass the parser function instead?